src/cogent3/core/annotation.py

```diff
@@ -96,61 +194,54 @@ def __getitem__(self, index):
     def _mapped(self, map):
         raise NotImplementedError
 
-    def get_drawables(self):
-        """returns a dict of drawables, keyed by type"""
-        result = defaultdict(list)
-        for a in self.annotations:
-            result[a.type].append(a.get_drawable())
-        return result
-
     def add_annotation(self, klass, *args, **kw):
+        from cogent3.core.sequence import Sequence
```

Comment:
For all methods that are being flagged as being removed from the Sequence object, we need to refactor their tests.

For example, there all tests inside `test_alignment.py` that use `add_annotation`, e.g. `test_get_annotations_from_any_seq` which has uses like the following
```python
seqs.get_seq("seq1").add_annotation(Feature, "exon", "annotation1", [(3, 8)])
```
should be changed to the new method and name
```python
seqs.get_seq("seq1").add_feature("exon", "annotation1", [(3, 8)])
```